test-data/advisories/rhba-2024_11505.json

```diff
@@ -0,0 +1,1645 @@
+{
```

Comment:
Those files can get rather big and clog up the repository. Why not compress them?